searx/settings.yml

```diff
@@ -501,6 +501,11 @@ engines:
     shortcut: bc
     categories: music
 
+  - name: baidu
+    engine: baidu
+    shortcut: baidu
+    disabled: false
```

Comment:
> Can we disable this engine by default @return42? 

@vojkovic isn't it disable .. here in the commit it is disabled.

Reply:
`disabled: false` means it's not disabled :)

Reply:
:facepalm: .. I always struggle with *double negative* .. can't believe that is positive in real live ;-)  

I send PR:

- https://github.com/searxng/searxng/pull/4465